### DISCUSSION TITLE
Installing filters - best practice for file locations?

### DISCUSSION THREAD

**Discussion by AndyClifton** [jgm/pandoc#10210] [Q&A]
I've written a filter for pandoc that I would like to use in multiple different file conversions. 

Currently I place the filter in the same directory as the source files, and then call pandoc using something like this:

```shell
$> pandoc ...  --filter filters/name_of_filter
```

where `filters` is a subdirectory.

I'd like to just maintain a single copy of my filters. 

Q: Does pandoc have a default location where it checks for filters where I could deploy my filter? I guess I can choose this myself by giving an absolute path to the filter, but is there a best practice for this?

**Comment by jpcirrus** [comment]
From the manual:

> In order of preference, pandoc will look for filters in
    1. a specified full or relative path,
    2. $DATADIR/filters where $DATADIR is the user data directory (see [--data-dir](https://pandoc.org/MANUAL.html#option--data-dir), above).

An addition, for executable (non-Lua) filters:

> 3. $PATH (executable only).
